source/_integrations/here_travel_time.markdown

```diff
@@ -6,7 +6,7 @@ ha_category:
   - Transport
   - Sensor
 ha_iot_class: Cloud Polling
-ha_release: '0.100'
+ha_release: '0.108'
```

Comment:
The `ha_release` indicates the initial introduction of the integration, it does not update.

Reply:
🤦‍♂ Thanks!

Reply:
Thanks!